src/Forms/DateField.php

```diff
@@ -101,6 +101,7 @@ class DateField extends TextField
      * to detect invalid values.
      *
      * @var mixed
+     * @deprecated 5.4.0 Will be removed without equivalent functionality to replace it
```

Comment:
```suggestion
     * @deprecated 5.4.0 Use $value instead
```
Presumably if people want to get the value, they should use the value property. Or the getValue() method.
Please apply this to the other `rawValue` deprecation phpdocs as well.

Reply:
I think it's correct as is as $this->value is still getting modified in CMS 5 - as mentioned on https://github.com/silverstripe/silverstripe-framework/pull/11449#discussion_r1839366435 this property has a specific purpose

Reply:
That comment mentions the past tense, and https://github.com/silverstripe/silverstripe-framework/pull/11449#discussion_r1839364649 says that `getValue()` should return the value "as unmodified as possible". Which I took to mean that the `value` property would now be the raw value, and the various methods for getting the value are where any modification of the value would take place.

Is there no way to get just _the_ value that was set to the field now?

Reply:
In the context of Date/Datetime/TimeField, $this->value will get modified after calling setValue($value). $this->rawValue is explicitly not getting modified at any point. There's no equivalent functionality in CMS 5.

My intention, at least, in CMS 6 at least is to not modify $this->value after setting it so that when validation is run it's running on the value that was set by the developer, though in practice Fields could still alter $this->value.

tl;dr Will be removed without equivalent functionality to replace it :-)

Reply:
Discussed in person - `$value` is functionally equivalent to `$rawValue` even if the exact value isn't identical

Reply:
Updated